JAICore/jaicore-search/src/test/java/ai/libs/jaicore/search/algorithms/standard/mcts/BRUETester.java

```diff
@@ -1,12 +1,12 @@
 package ai.libs.jaicore.search.algorithms.standard.mcts;
 
-import ai.libs.jaicore.search.algorithms.mdp.mcts.MCTSFactory;
+import ai.libs.jaicore.search.algorithms.mdp.mcts.MCTSBuilder;
 import ai.libs.jaicore.search.algorithms.mdp.mcts.brue.BRUEFactory;
 
 public class BRUETester extends MCTSForGraphSearchTester {
 
 	@Override
-	public <N, A> MCTSFactory<N, A> getFactory() {
+	public <N, A> MCTSBuilder<N, A, ?> getFactory() {

```

Comment:
Rename this method from "getFactory" to getBuilder maybe?

Reply:
I named it back to Factory; it's the correct usage of the Factory pattern here.